### DISCUSSION TITLE
Design ideas for score statistics breakdown

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#20333] [Ideas]
I suddenly thought of this idea and wanted to try to make it.
I don't know if the current in-game design is final, but I am aware that there may be more statistics in the future.
The ideas are various here, but the main one is to keep the score panel in the centre of the screen. The design tries to take the sense of "breakdown" a bit literally by using lines to connect statistics to the score panel.
The font size and other things might turn out to be small… again, it's just a concept. I hope you can take inspiration from these:

<details><summary><b>Example 1</b></summary>

![info example-0](https://user-images.githubusercontent.com/43073074/190521940-5ea065c7-b9f1-4e87-be53-09a8babf9d76.png)

</details>
<details><summary><b>Example 2</b></summary>

![info example-1](https://user-images.githubusercontent.com/43073074/190521944-224ca59f-cdde-4e3f-8fb7-f19e0db53220.png)

</details>
<details><summary><b>Example 3</b></summary>

![info example-2](https://user-images.githubusercontent.com/43073074/190521955-d89df23e-6434-4c44-a467-744b5f6b0cf3.png)

</details>

Regarding the size, I can imagine that the user could click on a statistic to expand it (perhaps showing additional information?) for example from a phone:

<details><summary><b>Expanded stats (1)</b></summary>

![info example-3](https://user-images.githubusercontent.com/43073074/190523994-c545e213-0944-4fd3-8dc6-5e6d6514f059.png)

</details>

<details><summary><b>Expanded stats (2)</b></summary>

![info example-4](https://user-images.githubusercontent.com/43073074/190524013-c90c9ccd-1045-4453-80c1-0bb8b2c2ab11.png)

</details>

**Comment by peppy** [comment]
@arflyte have you looked into the results screen with regards to adding online statistics tracking as described previously? (https://github.com/ppy/osu/issues/18877)

I don't think any further design changes should happen until that is figured out, since it is high priority and may change the overall layout of the results screen.
